integration-tests/contract-originate-token-contract.spec.ts

```diff
@@ -28,6 +28,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
       expect(op.hash).toBeDefined();
       expect(op.includedInBlock).toBeLessThan(Number.POSITIVE_INFINITY)
       contract = await op.contract();
+      console.log("contract address : "+contract.address)
```

Comment:
I think you forgot to remove the console.log here

Reply:
fixed